### ISSUE TITLE
OVER MSPT! 300+ [latest][sub-server for bungee]

### ISSUE THREAD

**Issue by GG-MD** [SkinsRestorer/SkinsRestorerX#1791]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using forge?

No

### Installed conforming to our guide?

- [X] I have read the installation guide and performed all steps

### Describe your issue

spark: https://spark.lucko.me/51JaKLAwkk

### Minecraft implementation

```markdown
- Proxy: NullCordX latest DEV
- Backend: PurPur 1.21.1 latest
```

### SkinsRestorer Version

spark profiler open...

### List of plugins

spark profiler open...

### Java Version

spark profiler open..

### Does your issue require a plugin in order to reproduce?

None

**Comment by xknat** [comment]
@AlexProgrammerDE can you pick this ine up?

**Comment by AlexProgrammerDE** [comment]
Was already fixed in dev builds afaik

**Comment by AlexProgrammerDE** [comment]
Released in 15.4.3
